### PR TITLE
Inertia web settings page

### LINKED ISSUE #247: Inertia Web Settings Page
Develop the settings page.

This should be fairly straightforward once #242 is completed. Use mock data, as we will connect to redux store in a separate task.

screen shot 2018-06-13 at 10 47 21 pm

### PR DESCRIPTION
**Pull request by rogermyang**
:tickets: **Ticket(s)**: Closes #247 

---

## :construction_worker: Changes
settings

disabled shutdown button for now.
## :flashlight: Testing Instructions

point to http://localhost:7900/web/#/home/settings

### CODE REVIEW

File: daemon/web/pages/settings/Settings.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          <TableHeader>
+            <TableRow>
+              <TableCell style={{ fontWeight: '900', fontSize: '14px' }}>Project-name</TableCell>
+              <TableCell />
+              <TableCell />
+            </TableRow>
+          </TableHeader>
+
+          <TableBody>
+            <TableRow>
```
Comment (bobheadxi) [inline daemon/web/pages/settings/Settings.js]: Sure you can list out the divs like this, but think about what the API will returns to the web app:

https://github.com/ubclaunchpad/inertia/blob/master/common/types.go#L4

```go
type DeploymentStatus struct {
	InertiaVersion       string   `json:"version"`
	Branch               string   `json:"branch"`
	CommitHash           string   `json:"commit_hash"`
	CommitMessage        string   `json:"commit_message"`
	BuildType            string   `json:"build_type"`
	Containers           []string `json:"containers"`
	BuildContainerActive bool     `json:"build_active"`
}
```

maybe there's a cleaner way to generate this table?

File: daemon/web/pages/main/Main.js [hunk 2]
```diff
@@ -232,6 +233,7 @@ class MainWrapper extends React.Component {
 
           <Link to={`${this.props.match.url}/dashboard`}>Click to go to Dashboard</Link>
           <Link to={`${this.props.match.url}/containers`}>Click to go to Containers</Link>
+          <Link to={`${this.props.match.url}/settings`}>Click to go to Settings</Link>
```
Comment (bobheadxi) [inline daemon/web/pages/main/Main.js]: 🔥

File: daemon/web/components/IconHeader/IconHeader.js [hunk 3]
```diff
@@ -9,8 +9,8 @@ const icons = {
   settings: <i className="fas fa-cog" />,
 };
 
-const IconHeader = ({ title, type }) => (
-  <div className="iconheader">
+const IconHeader = ({ title, type, style }) => (
+  <div className="iconheader" style={style}>
```
Comment (bobheadxi) [inline daemon/web/components/IconHeader/IconHeader.js]: nice!

File: daemon/web/pages/settings/Settings.js (hunk 1 above)
Comment (rogermyang) [reply daemon/web/pages/settings/Settings.js]: right the html is on the redundant side as i was trying to get a more consistent look. I'll get back to this

**Pull request merged**